pandas/io/formats/printing.py

```diff
@@ -56,12 +57,8 @@ def justify(texts, max_len, mode="right"):
         return [x.rjust(max_len) for x in texts]
 
 
-def _join_unicode(lines, sep=""):
-    try:
-        return sep.join(lines)
-    except UnicodeDecodeError:
-        sep = str(sep)
-        return sep.join([x.decode("utf-8") if isinstance(x, str) else x for x in lines])
+def _join_unicode(lines: Iterable[str], sep: str = "") -> str:
+    return sep.join(lines)
```

Comment:
Likely no longer needed?

Reply:
only used in two places. inline and remove?